src/locales/es/translation.json

```diff
@@ -270,6 +270,7 @@
     "pending": "Pendiente",
     "no_pending_invites": "No tienes invitaciones pendientes",
     "no_blocked_users": "No has bloqueado a ningún usuario",
-    "friend_code_copied": "Código de amigo copiado"
+    "friend_code_copied": "Código de amigo copiado",
+    "undo_friendship_modal_text": "This will undo your friendship with {{displayName}}"
```

Comment:
This added string is in English, is this correct?

Reply:
I was checking and yes, I translated and then I synced the fork making it revert the changes without noticing, let me change it real quick